boa/interpret.py

```diff
@@ -126,4 +126,11 @@ def from_etherscan(
     return ABIContractFactory.from_abi_dict(abi, name=name).at(addr)
 
 
+def from_etherscan_abi(
```

Comment:
i'm not sure why this function is needed

Reply:
i removed it in [82f6f7d](https://github.com/vyperlang/titanoboa/pull/122/commits/82f6f7d1a87f737f04ea1be1d4ac8c0342fa0c11). we can always add back alter